doc/05-context.md

```diff
@@ -18,12 +18,17 @@ with `Castor\Context`:
 ```php
 use Castor\Context;
 
+use function Castor\get_context;
+
 #[AsTask]
```

Comment:
First sentence of the chapter need to be updated too

Reply:
Thanks